src/main/java/mate/academy/bookshop/service/AuthenticationService.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.bookshop.service;
```

Comment:
It's better to place it in `security` package.

Reply:
still exist